specification/maps/data-plane/Search/preview/2021-11-01-preview/geocoding.json

```diff
@@ -65,7 +65,7 @@
     "Accept-Language": {
       "name": "Accept-Language",
       "in": "header",
-      "description": "Language in which search results should be returned. \n\nPlease refer to [Supported Languages](https://docs.microsoft.com/en-us/bingmaps/rest-services/common-parameters-and-types/supported-culture-codes) for details.",
+      "description": "Language in which search results should be returned. \n\nPlease refer to [Supported Languages](https://learn.microsoft.com/en-us/azure/azure-maps/supported-languages) for details.",
```

Comment:
Please do not have locale in URL. Link should be locale neutral for i18n.

Reply:
Updated all links to remove the locale